aws/aws-explanation/canonical-offerings.rst

```diff
@@ -39,7 +39,7 @@ The different variations and the means to find them in AWS are summarised below:
      - :doc:`CLI<../aws-how-to/instances/find-ubuntu-images>`, `Marketplace <https://aws.amazon.com/marketplace/search/results?searchTerms=Ubuntu+minimal&CREATOR=565feec9-3d43-413e-9760-c651546613f2&AMI_ARCHITECTURE=arm64&filters=CREATOR%2CAMI_ARCHITECTURE>`__
    
    * - Pro
-     - :doc:`CLI<../aws-how-to/instances/find-ubuntu-images>`, `Marketplace <https://aws.amazon.com/marketplace/search/results?searchTerms=Ubuntu+pro&CREATOR=565feec9-3d43-413e-9760-c651546613f2&AMI_ARCHITECTURE=x86_64&filters=CREATOR%2CAMI_ARCHITECTURE>`__
```

Comment:
i think we need to update the corollary arm link too

Reply:
Yup, I have updated it in the suggestion below.

Reply:
@rpocase I updated the arm listing